spec/model/user_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { User.new(name: 'Lilly', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Poland.', posts_counter: 0) }
+
+  before { subject.save }
+
+  it 'name should be present' do
+    subject.name = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'post counter should be equal or great than 0' do
+    subject.posts_counter = -1
+    expect(subject).to_not be_valid
+  end
+
+  it 'Subject should be valid' do
+    expect(subject).to be_valid
+  end
+
+  it 'Returns 3 most recent post' do
```

Comment:
- [ ] At that point we have one failing test. Please fix the issue and resubmit it again. Also, if the app is working on your side try to reset your database with `rails db:reset` and run the test again. After that if the tests are working smoothly you are free to leave your screenshot below. Thank you. This is the issue we are facing:

<img width="720" alt="Screenshot 2023-02-24 at 18 05 52" src="https://user-images.githubusercontent.com/69011963/221186440-acc3b071-2f14-4d27-adeb-4b1380e6fd22.png">